src/PageContent/Settings/OrganizationDetails.js

```diff
@@ -46,6 +98,12 @@ class OrganizationDetails extends Component {
                             : 
                             <span />
                         }
+                        {this.props.accountType === AccountType.RECEIVING_AGENCY ? 
+                            <h6>Availabilities: {this.convertTimes()}</h6> 
+                            
+                            : 
+                            <span />
```

Comment:
A slightly cleaner way of doing this where you don't have to put an empty span is to use && instead like this: 
```suggestion
                        {this.props.accountType === AccountType.RECEIVING_AGENCY && 
                            <h6>Availabilities: {this.convertTimes()}</h6> 
                         }
```